relay-log/build.rs

```diff
@@ -21,6 +24,43 @@ fn emit_release_var() -> Result<(), io::Error> {
     Ok(())
 }
 
+fn list_crates() -> Vec<String> {
```

Comment:
Do we want this `fn` rather return `io::Result` instead of unwrapping everything? Since it's called in `emit_cratt_list` which actually properly returns result.  I think it can be used then to return nicer error messages.

Reply:
Good point, I was thinking about this. 

The current old build script swallows errors silently, so I chose to expose these errors with a panic. Probably, an error is even better, but we would need additional context for why it panics. I'll follow up about this.